subprojects/plugin-development/src/main/java/org/gradle/plugin/devel/tasks/ValidateTaskProperties.java

```diff
@@ -105,14 +110,24 @@
  * @since 3.0
  */
 @CacheableTask
-@SuppressWarnings("WeakerAccess")
-public class ValidateTaskProperties extends ConventionTask implements VerificationTask {
-    private FileCollection classes;
-    private FileCollection classpath;
-    private RegularFileProperty outputFile = getProject().getObjects().fileProperty();
+@SuppressWarnings("deprecation")
```

Comment:
Why is this needed?

Reply:
Since I marked the interface which causes to create the bridge methods as deprecated.

Reply:
Does this mean that if I use `ValidateTaskProperties` in my code it will be marked as deprecated, too?

Reply:
I tried and I didn't get any deprecation warnings. Not for extending the class nor for calling `getClasses()`.